### PR TITLE
[RA-1 Ch07] Typo's, spelling, language corrections

### PR DESCRIPTION
**Pull request by pgoyal01**
Made corrections discovered during creation of GSMA doc

Partially Fixes Issue #2618

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 1]
```diff
@@ -21,21 +23,23 @@
...
 
 The provisioning of the infrastructure is typically performed by provisioning tools while the deployment of the software and the configuration of the software, and where needed the infrastructure, falls in the domain of configuration management tools. A single tool may support both provisioning and configuration management.
 
-Operators may choose certain paradigms with respect to how they provision and configure their IaaS cloud. These paradigms will drive the selection of the provisioning and configuration tools. In this chapter we will discuss the capabilities of provisioning and configuration management systems; some Open Source tools may be mentioned but their capabilities are beyond the scope of this chapter. 
+Operators may choose certain paradigms with respect to how they provision and configure their IaaS cloud. These paradigms will drive the selection of the provisioning and configuration tools. In this chapter we will discuss the capabilities of provisioning and configuration management systems; some open-source tools may be mentioned but their capabilities are beyond the scope of this chapter.
```
Comment (CsatariGergely) [inline doc/ref_arch/openstack/chapters/chapter07.md:30]:
```suggestion
Operators may choose certain paradigms with respect to how they provision and configure their IaaS cloud. These paradigms will drive the selection of the provisioning and configuration tools. In this chapter we will discuss the capabilities of provisioning and configuration management systems; some open source tools may be mentioned but their capabilities are beyond the scope of this chapter. 
```

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 2]
```diff
@@ -142,15 +146,15 @@ software to watch for changing conditions.
...
 
 
 <p align="center"><img src="../figures/RA1-Ch07-Monitoring-Logging-Framework.png" alt="Monitoring and Logging Framework"><b>
   Figure 7-1: Monitoring and Logging Framework</b> </p>
 
 The monitoring and logging framework (**Figure 7-1**) leverages Prometheus as the monitoring engine and 
-Fluentd for logging. In addition, the framework uses Elasticsearch to store and organize logs for easy access. 
-Prometheus agents pull information from individual components on every host.  Fluentd, an Open Source data 
+Fluentd for logging. In addition, the framework uses Elasticsearch to store and organise logs for easy access. 
+Prometheus agents pull information from individual components on every host.  Fluentd, an open-source data
```
Comment (CsatariGergely) [inline doc/ref_arch/openstack/chapters/chapter07.md:157]:
```suggestion
Prometheus agents pull information from individual components on every host.  Fluentd, an open source data 
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md:30]: @CsatariGergely here open source is an adjective to tools and hence the hyphenation.

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md:157]: @CsatariGergely here open source is an adjective to data and hence the hyphenation.

**Review by karinesevilla — APPROVED** [review]

**Review by ildikov — APPROVED** [review]

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md:30]: @CsatariGergely Please respond or resolve - Thanks

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md:157]: @CsatariGergely Please respond or resolve - Thanks

**Review by walterkozlowski — APPROVED** [review]

**Pull request closed without merging**